frontend/src/pages/RunDetails.tsx

```diff
@@ -626,7 +626,7 @@ class RunDetails extends Page<RunDetailsInternalProps, RunDetailsState> {
                     <div>
                       <DetailsTable
                         title='Run parameters'
-                        fields={workflowParameters.map(p => [p.name, p.value || ''])}
```

Comment:
Why deleting `|| ''`?

Reply:
The main reason why I deleted this is trying to fix e2e test, though it's not the root cause. However, after checking the code base, I found that in the `Parameter` definition, `value` is an optional field. It means that we don't have to put `''` as a default input, so I didn't add it back.

Ref: https://github.com/kubeflow/pipelines/blob/master/frontend/src/third_party/mlmd/argo_template.ts#L212-L231